test/testSimplifier.ts

```diff
@@ -162,6 +162,9 @@ function tokenizeKeySequence(sequence: string): string[] {
 
     if (char === '>') {
       isBracketedKey = false;
+      if (key.startsWith("<c-")) {
+        key = "ctrl+" + key.substring(3, key.length - 1);
```

Comment:
so, this is pretty gross, and i'm not sure if we want people to be able to turn `<c-abc>` into `ctrl+abc`. but at least it removes the `TODO hack` from the other place.